portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,16 +44,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      int id = this.comments.size();
+      //TODO: use GSON deserialization
       String name = request.getParameter("name");
       String content = request.getParameter("content");
-      Comment comment = new Comment(id, name, content);
-      this.comments.add(comment);
+      Comment comment = new Comment(0, name, content); //Since the ID will only be assigned by the Datastore, I set it to 0. 
+      //What would be a better practice?
```

Comment:
Use null to indicate "lack of the value". IIRC you cannot assign null to `long`, but you can do it with `Long`.
Alternatively, make interface accept author and content - without creating a stub comment temporarily.

Reply:
Dominic's suggestion is even better!